### DISCUSSION TITLE
Dokumentation der DB-Felder

### DISCUSSION THREAD

**Discussion by Larian77** [Larian77/Openbiblio#20] [General]
Wir sollten hier mal dokumentieren welche DB-Einträge noch dazu gekommen sind bzw. welche DB-Strukturen geändert wurden, damit wir in der nächsten Version auch eine Update-Funktion mal wieder hinzufügen

We should document here which DB entries have been added or which DB structures have been changed so that we can add an update function again in the next version

**Comment by Larian77** [comment]
Was mir einfällt:

bei den inserts müsste man nur schauen wie man das Sprachabhängig steuert?

0.8.0:
insert into biblio_status_dm values ('pre','Präsenz','N'); / insert into %prfx%biblio_status_dm values ('pre','Presence','N'); 

eine meiner ersten Versionen / one of my first versions::
insert into %prfx%usmarc_indicator_dm values (901,1,'#','Undefiniert'); / insert into %prfx%usmarc_indicator_dm values (901,1,'#','Undefined');
insert into %prfx%usmarc_indicator_dm values (901,2,'#','Undefiniert'); / insert into %prfx%usmarc_indicator_dm values (901,2,'#','Undefined');
insert into %prfx%usmarc_indicator_dm values (902,1,'#','Undefiniert'); / insert into %prfx%usmarc_indicator_dm values (902,1,'#','Undefined');
insert into %prfx%usmarc_indicator_dm values (902,2,'#','Undefiniert'); / insert into %prfx%usmarc_indicator_dm values (902,2,'#','Undefined'); 
insert into %prfx%usmarc_subfield_dm values (901,'a','min. Spieler','N'); / insert into %prfx%usmarc_subfield_dm values (901,'a','min. Player','N');
insert into %prfx%usmarc_subfield_dm values (901,'b','max. Spieler','N'); / insert into %prfx%usmarc_subfield_dm values (901,'b','max. Player','N');
insert into %prfx%usmarc_subfield_dm values (901,'c','min. Alter','N'); / insert into %prfx%usmarc_subfield_dm values (901,'c','min. Age','N'); 
insert into %prfx%usmarc_subfield_dm values (901,'d','max. Alter','N'); / insert into %prfx%usmarc_subfield_dm values (901,'d','max. Age','N');
insert into %prfx%usmarc_subfield_dm values (901,'e','Spieldauer','N'); / insert into %prfx%usmarc_subfield_dm values (901,'e','Duration of the Game','N');
insert into %prfx%usmarc_subfield_dm values (902,'a','Bild des Mediums','N'); / insert into %prfx%usmarc_subfield_dm values (902,'a','Bibliography Picture','N');
insert into %prfx%usmarc_tag_dm values (9,901,'Spieldaten (bei Spielen)','Undefiniert','Undefiniert','Y'); / insert into %prfx%usmarc_tag_dm values (9,901,'Fields for Games','Undefined','Undefined','Y');
insert into %prfx%usmarc_tag_dm values (9,902,'Zusätzliche Daten','Undefiniert','Undefiniert','Y'); / insert into %prfx%usmarc_tag_dm values (9,902,'Additional Fields','Undefined','Undefined','Y');

ebenfalls früher / earlier as well:
Spalte mbrshipend in member-Tabelle, es gab nie einen Updateprozess der die Spalte ergänzt, wenn nicht vorhanden / Column mbrshipend in member table, there was never an update process that added the column if not present

Die Renew-Spalten die ich eingefügt habe wurden schon mit einem Update (auf 0.5.2) hinzugefügt / The renew columns I added were already added with an update (to 0.5.2)

**Comment by 375michael40veit** [comment]
### Version 0.8.1
**table member**
_Create columns:_
- mbrshipend
- pwd
- pwd_timeout
- pwd_forgotten
- pwd_forgotten_time

**table staff**
_Create columns_
- pwd_timeout
- pwd_forgotten
- pwd_forgotten_time

_Modify column_

- pwd 

**table settings**
_Create columns_

- login_attempts
- pwd_timeout
- mbraccount_online

**Install and update tables**

- email_settings
- email_mesages

You can find more details in the file classes/UpgradeQuery.php.
